target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-mf286d.dts

```diff
@@ -87,6 +87,7 @@
 
 		ess-psgmii@98000 {
 			status = "okay";
+			/delete-node/ ethernet-phy@0;
```

Comment:
Drop this, this node does not exist anymore at all.

Reply:
Done.